### ISSUE TITLE
Cant connect to the Nanoleaf

### ISSUE THREAD

**Issue by genesfa** [rowak/nanoleaf-aurora#1]
Hey there i wrote a code with this API a month ago and it worked like i expected.

Now i want to run it again and it dont work because the    List<AuroraMetadata> auroras = Setup.findAuroras(timeout); line don't find my nanoleafe in the network but they are both in the same network and are connected to wlan.

Do you know the problem ?

**Comment by amybytes** [comment]
It seems you're right. The code was always a bit unreliable, but it still seems to be up-to-date with the Nanoleaf documentation, and I can't seem to find anything wrong with it.

I've tested the [mDNS](https://forum.nanoleaf.me/docs/openapi#_fk6r6vggacdj) discovery method and I'm currently working on replacing the old findAuroras() method with this one. If you want to test it out for yourself, add [this jar](https://github.com/fflewddur/hola/releases/download/v0.2.1/hola-0.2.1.jar) and the following dependency to a blank maven project:
```
<dependency>
  <groupId>org.slf4j</groupId>
  <artifactId>slf4j-simple</artifactId>
  <version>1.6.1</version>
</dependency>
```
Then run the following code in the same project:
```java
import java.util.Set;

import net.straylightlabs.hola.dns.Domain;
import net.straylightlabs.hola.sd.Instance;
import net.straylightlabs.hola.sd.Query;
import net.straylightlabs.hola.sd.Service;

public class MDNSTest {
	public static void main(String[] args) {
		try {
			Service service = Service.fromName("_nanoleafapi._tcp");
	        	Query query = Query.createFor(service, Domain.LOCAL);
	        	Set<Instance> instances = query.runOnce();
	        	instances.stream().forEach(System.out::println);
		}
		catch (UnknownHostException e) {
			System.out.println("Unknown host: " + e);
		}
		catch (IOException e) {
			System.out.println("IO error: " + e);
		}
	}
}
```
Upon running you should see information about all of the Nanoleaf devices on your local network.

**Comment by amybytes** [comment]
Should be fixed in 78a14c70eb56fd756c90f30a448850f4813eb96d and much faster too. Also released v1.5.1 (update your maven/jar dependency). Let me know if everything in the new release is working properly.

**Comment by genesfa** [comment]
Hey thanks for the response i use now the findAuroras() (the function without timeout) function and it works perfekt.

Thank you
